xml/article_nfs_storage.xml

```diff
@@ -450,255 +465,240 @@ include /etc/drbd.d/*.res;</screen>
-      on the same node on which the DRBD promotable clone resource is in the
-      primary role:
+      Add a colocation constraint to make sure that the two resources
+      always start on the same node:
      </para>
-<screen>&prompt.crm.conf;<command>order o-drbd_before_nfs Mandatory: \
-  clone-drbd_nfs:promote g-nfs:start</command>
-&prompt.crm.conf;<command>colocation col-nfs_on_drbd inf: \
-  g-nfs clone-drbd_nfs:Promoted</command></screen>
-    </listitem>
-    <listitem>
+<screen>&prompt.crm.conf;<command>colocation co_nfs-share-with-nfs-state \
+  inf: fs_nfs-share fs_nfs-state</command></screen>
+    </step>
+    <step>
+     <para>
+      Add colocation constraints to make sure that each resource always starts
+      on the node where the related DRBD promotable clone is in the primary role:
+     </para>
+<screen>&prompt.crm.conf;<command>colocation co_nfs-share-on-drbd \
+  inf: fs_nfs-share cl_nfs-share:Promoted</command>
+&prompt.crm.conf;<command>colocation co_nfs-state-on-drbd \
+  inf: fs_nfs-state cl_nfs-state:Promoted</command></screen>
+    </step>
+    <step>
+     <para>
+      Add order constraints to make sure the DRBD promotable clones always start
+      before the file system resources:
+     </para>
+<screen>&prompt.crm.conf;<command>order o_drbd-before-nfs-share \
+  Mandatory: cl_nfs-share:promote fs_nfs-share:start</command>
+&prompt.crm.conf;<command>order o_drbd-before-nfs-state \
+  Mandatory: cl_nfs-state:promote fs_nfs-state:start</command></screen>
+    </step>
+    <step>
      <para>
       Commit this configuration:
      </para>
 <screen>&prompt.crm.conf;<command>commit</command></screen>
-    </listitem>
-   </orderedlist>
+    </step>
+   </procedure>
    <para>
-    After these changes have been committed, Pacemaker mounts the DRBD device
-    to <filename>/srv/nfs/work</filename> on the same node. Confirm this with
-    <command>mount</command> (or by looking at <filename
-     >/proc/mounts</filename>).
+    Pacemaker mounts <literal>/dev/drbd1</literal> to <filename>/srv/nfs/share</filename>,
+    and <literal>/dev/drbd2</literal> to <filename>/var/lib/nfs</filename>. Confirm this
+    with <command>mount</command>, or by looking at <filename >/proc/mounts</filename>.
    </para>
   </sect2>
 
-  <sect2 xml:id="sec-ha-quick-nfs-resources-nfsexport">
-   <title>NFS export resources</title>
+  <sect2 xml:id="sec-ha-quick-nfs-resources-nffserver-exportfs-vip">
+   <title>Creating NFS supporting resources</title>
    <para>
-    When your DRBD, LVM, and file system resources are working properly,
-    continue with the resources managing your NFS exports. To create highly
-    available NFS export resources, use the <literal>exportfs</literal>
-    resource type.
+    Finally, create cluster resources for the NFS server daemon, the NFS exports,
+    and the virtual IP address. <emphasis>Do not</emphasis> commit this configuration
+    until after you add the colocation and order constraints.
```

Comment:
Probably one more prerequisite to info user to run, just in case, 
`zypper in nfs-kernel-server`